osu.Game.Rulesets.Osu/Difficulty/OsuPerformanceCalculator.cs

```diff
@@ -210,8 +210,7 @@ private double computeAimValue(ScoreInfo score, OsuDifficultyAttributes attribut
                 aimValue *= 1.3 + (totalHits * (0.0016 / (1 + 2 * effectiveMissCount)) * Math.Pow(accuracy, 16)) * (1 - 0.003 * attributes.DrainRate * attributes.DrainRate);
             else if (score.Mods.Any(m => m is OsuModTraceable))
             {
-                // We want to give more reward for lower AR when it comes to aim and HD. This nerfs high AR and buffs lower AR.
-                aimValue *= 1.0 + 0.04 * (12.0 - approachRate);
+                aimValue *= 1.0 + OsuDifficultyCalculator.CalculateVisibilityBonus(score.Mods, approachRate);
```

Comment:
We need to not forget to remove this when realtime goes live @tsunyoku

Reply:
By default nothing will happen when real time goes live, since I made sure the bonuses for TC don't exist in star rating. Something to follow up with once real time happens though